web/src/testdata.test.tsx

```diff
@@ -49,7 +49,7 @@ function getMockRouterProps<P>(data: P) {
 }
 
 function oneResource(): any {
-  const ts = Date.now().toLocaleString()
+  const ts = Date.now().valueOf()
```

Comment:
it turns out `.toLocaleString` is inherited from `number`, and returns a human-readable formatting of the number of milliseconds since epoch, which looks like `1,482,363,367,071`.

timeago passes that to `Date.parse`, which rejects it because of the commas, and then kind of assumes that since there are at least three non-contiguous numbers (i.e., `/\d+/` in the string), the second one must be a month, which means it needs to have one subtracted to make it 0-based, and then reconcatenates them ([code](https://github.com/nmn/react-timeago/blob/69b34a295f3c38e53064ce084767c7c294e8ef04/src/dateParser.js#L13)), which is why the old snapshot timestamp was 1942 and timeago rendered it as 75yr